common/build/eslint-config-fluid/eslint7.js

```diff
@@ -4,39 +4,37 @@
  */
 
 module.exports = {
-    "env": {
-        "browser": true,
-        "es6": true,
-        "node": true
+    env: {
```

Comment:
I think we should use 2-space indentation for JSON. I bet the default is coming from the .editorconfig, so maybe change it there or remove editorconfig altogether?

Reply:
Updated.